Update Notebooks

alanlujan91 opened the pull request:
Run notebooks so static file is rendered correctly. Although interactive notebook runs on mybinder, the initial rendition shows an error due to local environment inconsistencies.

alanlujan91 committed:
update notebooks

run notebooks so static file is rendered correctly

llorracc committed:
Delete blog_post-checkpoint.ipynb

llorracc reviewed (commented):
Could you add checkpoints and pyc files to the gitignore?

alanlujan91 committed:
update gitignore

added standard python gitignore from https://github.com/github/gitignore/blob/master/Python.gitignore

alanlujan91 commented:
@llorracc I added the standard python gitignore template from https://github.com/github/gitignore/blob/master/Python.gitignore
